index.html

```diff
@@ -1,28 +1,101 @@
-<!DOCTYPE html>
-<html lang="en">
+<html>
+    <head>
+        <link href="css/style.css" rel="stylesheet" >
```

Comment:
Meta tags should go here. They help give more description to your site

Reply:
- [ ] t's better to be more polite you can say _Kindly use this..._ _or it's better to..._

Reply:
Thanks for the feedback

Reply:
Add your updated comments here.